tensorflow/core/common_runtime/bfc_allocator.h

```diff
@@ -372,7 +372,7 @@ class BFCAllocator : public VisitableAllocator {
   size_t BinNumToSize(BinNum index) {
     return static_cast<size_t>(256) << index;
   }
-  BinNum BinNumForSize(size_t bytes) {
+  inline BinNum BinNumForSize(size_t bytes) {
```

Comment:
This change has no effect because we're in a class definition. Please revert this line.

Reply:
Thank you very much for your suggestion.